src/data/actions/grades.js

```diff
@@ -132,14 +132,14 @@ const fetchPrevNextGrades = (endpoint, cohort, track) => (
 );
 
 
-const updateGrades = (courseId, updateData) => (
+const updateGrades = (courseId, updateData, searchText, cohort, track) => (
   (dispatch) => {
     dispatch(gradeUpdateRequest());
     return LmsApiService.updateGradebookData(courseId, updateData)
       .then(response => response.data)
       .then((data) => {
         dispatch(gradeUpdateSuccess(data));
-        dispatch(fetchGrades(courseId, null, null, true));
+        dispatch(fetchMatchingUserGrades(courseId, searchText, cohort, track));
```

Comment:
What if we did not have any search texts in the fields, would this still work?
Basically, if the grade adjustment operated on top of default all view, would this change cause breakage?

Reply:
it shouldn't break anything. it calls LmsApiService.fetchGradebookData which handles null arguments correctly

Reply:
Could you please test that case to make sure?

Reply:
seems to be fine!